packages/block-editor/src/date-time-picker/styles.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
These styles are the same as the styles in the `blocks` package.
We can avoid this duplication by creating a `react-datepicker` extension inside `blocks/src/components`.

Following the naming we already have there you'd create something like: `packages/blocks/src/components/form-date-picker/index.js`

```JavaScript
/**
 * External dependencies
 */
import DatePicker from 'react-datepicker';
import styled from '@emotion/styled';

//Feel free to move these styles to another file if you want
export const DatePickerWrapper = styled.div`
	input,
	textarea {
		color: var( --wp--preset--color--foreground );
		background-color: transparent;
		border-width: 1px;
		border-style: solid;
		padding: 8px;
	}

	.react-datepicker__day--selected {
		color: var( --wp--preset--color--background );
		background-color: var( --wp--preset--color--primary );
	}
`;

const FormDatePicker = ( props ) => {
	return (
		<DatePickerWrapper>
			<DatePicker
				dateFormat="MMMM d, yyyy"
				popperModifiers={ [
					{
						name: 'arrow',
						options: {
							padding: 100,
						},
					},
				] }
				{ ...props }
			/>
		</DatePickerWrapper>
	)
}

export default FormDatePicker;
```

That would allow you to also add some default values to properties like `dateFormat` and `popperModifiers`.
This way, you could use the `<FormDatePicker />` on both packages, `blocks` and `block-editor`

On `blocks/src/date-time-picker/index.js`:
```JavaScript
		<FormInputWrapper
			className={ classes }
			style={ { ...useColorStyles( attributes ) } }
		>
			<FormInputWrapper.Label className="crowdsignal-forms-date-time-picker-block__label">
				<RawHTML>{ attributes.label }</RawHTML>
			</FormInputWrapper.Label>
			<FormDatePicker
				selected={ parsedDate }
				onChange={ handleDateChange }
			/>
			{ error && <ErrorMessage>{ error }</ErrorMessage> }
		</FormInputWrapper>
```

On `block-editor/src/date-time-picker/edit.js`:
```JavaScript
		<FormInputWrapper
			style={ { ...useColorStyles( attributes ) } }
			className={ classes }
		>
			<Sidebar { ...props } />
			<FormInputWrapper.Label className="crowdsignal-forms-date-time-picker-block__label">
				<RichText
					placeholder={ __(
						'Enter date picker label',
						'block-editor'
					) }
					onChange={ handleChangeLabel }
					value={ attributes.label }
				/>
			</FormInputWrapper.Label>
			<FormDatePicker
				selected={ new Date() }
			/>
		</FormInputWrapper>
```

Reply:
Good stuff, I just pushed a commit with your suggestions.  I'll tackle the other CSS next